lib/github_changelog_generator/generator/generator_tags.rb

```diff
@@ -99,11 +99,11 @@ def detect_link_tag_time(newer_tag)
 
     # @return [Object] try to find newest tag using #Reader and :base option if specified otherwise returns nil
     def detect_since_tag
-      @since_tag ||= options.fetch(:since_tag) { version_of_first_item }
+      @detect_since_tag ||= options.fetch(:since_tag) { version_of_first_item }
     end
 
     def detect_due_tag
-      @due_tag ||= options.fetch(:due_tag, nil)
+      @detect_due_tag ||= options.fetch(:due_tag, nil)
```

Comment:
Could we take the bot advice and rename the method instead? This goes for all memoized methods in this change. 

Thank you for making this change!